recipes/reprit/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "reprit" %}
+{% set version = "0.3.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 1e5198c852589518814118fa72dfd0b975e6633fed4c41ec22d69e1824b3b458
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+
+test:
+  imports:
+    - {{ name }}
+
+about:
+  home: https://github.com/lycantropos/{{ name }}
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: 'Auto __repr__ method generation.'
+  description: 'Auto __repr__ method generation.'
+  doc_url: https://github.com/lycantropos/{{ name }}
```

Comment:
can it be a link to readthedocs? because `reprit` has [one](https://reprit.readthedocs.io/en/latest)

Reply:
Done